srcpkgs/Tuba/template

```diff
@@ -1,14 +1,21 @@
 # Template file for 'Tuba'
 pkgname=Tuba
-version=0.2.0
+version=0.3.2
 revision=1
 build_style=meson
 configure_args="-Ddistro=true"
-hostmakedepends="pkg-config gettext desktop-file-utils vala glib-devel"
-makedepends="libadwaita-devel libgee-devel libsecret-devel gtksourceview5-devel"
+hostmakedepends="desktop-file-utils gettext pkg-config"
+makedepends="gtksourceview5-devel libadwaita-devel libgee-devel libsecret-devel
+ libwebp-devel"
+depends="webp-pixbuf-loader"
+checkdepends="appstream-glib"
 short_desc="Browse the fediverse"
 maintainer="Michal Vasilek <michal@vasilek.cz>"
 license="GPL-3.0-only"
-homepage="https://tuba.geopjr.dev/"
-distfiles="https://github.com/GeopJr/Tuba/archive/refs/tags/v${version}.tar.gz"
-checksum=c698206bbbd06a1ce5a6e8cc9e6590411dc1cce1dde87fb0e34d8e35432ee2b1
+homepage="https://github.com/GeopJr/Tuba"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=1458b343f069efca00b0b5da49b9eb4b529e49de3d701da06d9529828be19cf9
+
+post_install() {
+	ln -s dev.geopjr.Tuba ${DESTDIR}/usr/bin/tuba
+}
```

Comment:
why?

Reply:
Easier to launch from the terminal. Saw it in the AUR template, liked it. Is this a bad practice?

Reply:
I am against this

Reply:
Removed.

Please clarify. Are you against this in your templates? Or are you (as a member of VoidLinux) against such practice in any templates?

Reply:
I am personally against us (a distribution) creating a symlink for a graphical program just so it is easier to launch from CLI. We have to choose the same symlink name as all other distributions, make sure it doesn't conflict with any other package and somehow solve conflicts in that case. If people want to have an alias, they can set it, but launching a graphical application should be usually done with a graphical launcher which parses uses standard .desktop files instead of plain commands.